snmp/fail2ban

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Addthis to snmpd.conf as below.
+# extend fail2ban /etc/snmp/fail2ban
+#
+# Also please verify your fail to ban instalation for proper table/chain names.
+
+if [ `uname` = "FreeBSD" ]; then
+	/sbin/pfctl -t fail2ban -T show | /usr/bin/grep -c .
+fi
+if [ `uname` = "Linux" ]; then
+	f2b1=`iptables -L -n | grep -c f2b\`
```

Comment:
I had to remove the \ to make it work and this just returns the number of chains, not the number of blocked users.